napi-inl.h

```diff
@@ -2015,9 +2015,16 @@ inline CallbackInfo::~CallbackInfo() {
   }
 }
 
+inline Value CallbackInfo::NewTarget() const {
+  napi_value newTarget;
+  napi_status status = napi_get_new_target(_env, _info, &newTarget);
+  return Value(_env, newTarget);
+}
+
 inline bool CallbackInfo::IsConstructCall() const {
-  bool isConstructCall;
-  napi_status status = napi_is_construct_call(_env, _info, &isConstructCall);
+  napi_value newTarget;
+  napi_status status = napi_get_new_target(_env, _info, &newTarget);
+  bool isConstructCall = (new_target != nullptr);
   NAPI_THROW_IF_FAILED(_env, status, false);
```

Comment:
Shouldn’t this line be part of `NewTarget()` as well?

By the way, what I had in mind was just replacing `IsConstructCall` with
```c++
inline bool CallbackInfo::IsConstructCall() const {
  return !NewTarget().IsEmpty();
}
```